dracut/modules.d/90kiwi-dump/kiwi-dump-image.sh

```diff
@@ -78,10 +80,9 @@ function get_disk_list {
     for disk_meta in $(
         eval lsblk "${blk_opts}" | grep -E "disk|raid" | tr ' ' ":"
     );do
+        kiwi_install_disk_part=$(eval lsblk "${blk_opts_plus_label}" | tr ' ' ":" | grep ":${kiwi_install_volid}$" | cut -f1 -d:)
```

Comment:
The change looks good to me, this is a better way to find the partition labeled with the install label. I think the tr call can be changed to

```bash
| tr -s ' ' : |
```
-s, --squeeze-repeats

Just in case there are more spaces playing tricks

Reply:
and it would be nice to break up the line to not exceed the 80characters... just a policy we follow for initrd shell code and because I'm an old man ;)

Reply:
Thank you! I'll update the change as indicated, and add the lsblk output and blkid output to the conversation when I get back into the office.

I also just realized, it's probably unnecessary to calculate the install partition on each iteration of the loop. Not that we care too much about performance in this particular loop, but it would technically perform better if it was calculated outside the loop.

Reply:
yep good point, makes perfect sense to me too